operator/src/main/kotlin/eu/glasskube/operator/config/ConfigService.kt

```diff
@@ -58,12 +58,14 @@ class ConfigService(
 
     private val dynamicCloudProvider
         get() = when {
+            kubernetesClient.configMaps().inNamespace("kube-system")?.withName("shoot-info")?.get()?.data?.get("extensions")?.contains("shoot-dns-service") == true ->
```

Comment:
```suggestion
            kubernetesClient.configMaps().inNamespace("kube-system").withName("shoot-info").get()?.data?.get("extensions")?.contains("shoot-dns-service") == true ->
```

as @kosmoz mentioned in his comment:

> I think the first result that can actually be null is that of get(), so you can omit the first two safe calls